api.planx.uk/modules/flows/publish/service/applicationTypes.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  ComponentType,
+  type FlowGraph,
+} from "@opensystemslab/planx-core/types";
+import { getValidSchemaValues } from "@opensystemslab/planx-core";
+import { isComponentType } from "../../flowHelpers.js";
+
+export const checkStatutoryApplicationTypes = (flattenedFlow: FlowGraph) => {
```

Comment:
nit: it'd be helpful to have a return type here! Eg `boolean`? "check" doesn't typically indicate a boolean return for me semantically!